src/main/java/edu/cornell/kfs/rass/batch/service/impl/RassUpdateServiceImpl.java

```diff
@@ -421,7 +427,11 @@ protected boolean propertyValuesMatch(String propertyName, Object oldBo, Object
             return GlobalVariables.doInNewGlobalVariables(
                     buildSessionForSystemUser(),
                     () -> createAndRouteMaintenanceDocumentInternal(businessObjects, maintenanceAction, objectDefinition));
-        } catch (Exception e) {
+        }
+        catch(ValidationException ve) {
+            throw new RuntimeException(ve.getMessage());
```

Comment:
It's not immediately clear that the caught ValidationException here has had its message forcibly constructed by the methods below. Is there some way to make that a little clearer (perhaps by using a different variable name), so that future change reviewers won't accidentally think that the MessageMap errors are being ignored by this error message setup?

Also, the "catch" block start should be on the same line as the closing curly brace, and there should be a space after the "catch" keyword.

Reply:
updated name